wg-code-quality/wg-charter.md

```diff
@@ -0,0 +1,59 @@
+# WG Code-Quality Charter
+
+## Scope
+
+The Code-Quality Working Group has two goals:
+
+* (short-term) **focus on the improvement of code quality** and
+* (long-term) **create an inter-SIG process for identifying, agreeing on and addressing code quality issues**
+
+The long-term goal is achieved when the process is implemented and executed as required.
+
+## In scope
+
+The main objective of this effort is to continuously improve the quality of the project codebase.
+
+With this in mind, there is a need to define a clear scope on which areas and through what means this can be accomplished.
+
+* Create and maintain code-quality guidelines, best practices and standards.
+* Assure distribution of knowledge in regard to code quality.
+  * This may include presentations, documentation, lectures, certifications, linters, etc.
+* Enforcement of selected standards.
+  * Through automation and active intervention in code reviews.
+  * Through codebase infrastructure. E.g reusable libraries.
+* Management:
+  * Prioritizing tasks.
+  * Focusing on selected topics.
+* Monitoring and identification of code-quality issues.
+
+The WG will therefore work on processes through which the above goals can be achieved.
```

Comment:
> work on processes

I am unsure if this is clear enough to keep it in the text.

As I see it, the WG should work on providing the means to improve the quality until a point where it is well integrated in the projects. The points above hints what we expect to have in the end.

I do not think it is limited to creating processes, it is there to facilitate the effort to create content, refactor code, etc. As it is already mentioned in the points above, I think this sentence is redundant.

Reply:
I have changed the In Scope section by changing the last sentence to what you pointed out before, which IMHO brings it down into an actionable goal. WDYT?